packages/fx-core/src/plugins/resource/apiconnector/plugin.ts

```diff
@@ -35,14 +42,61 @@ export class ApiConnectorImpl {
     const languageType: string = ctx.projectSetting!.programmingLanguage!;
     const config: ApiConnectorConfiguration = this.getUserDataFromInputs(inputs);
     for (const componentItem of config.ComponentPath) {
+      await this.scaffoldInComponent(projectPath, componentItem, config, languageType);
+    }
+    return ResultFactory.Success();
+  }
+
+  private async scaffoldInComponent(
+    projectPath: string,
+    componentItem: string,
+    config: ApiConnectorConfiguration,
+    languageType: string
+  ): Promise<ApiConnectorResult> {
+    const componentPath = path.join(projectPath, componentItem);
+    const backupFolderName = generateTempFolder();
+    const sampleFileName = getSampleCodeFileName(config.APIName, languageType);
```

Comment:
we do not need to backup the sample file, since we should throw an error if sample file already exists.

Reply:
sure